Sprint-1/fix/median.js

```diff
@@ -2,10 +2,24 @@
 // Start by running the tests for this function
 // If you're in the Sprint-1 directory, you can run `npm test -- fix` to run the tests in the fix directory
 
+// function calculateMedian(list) {
+//   const middleIndex = Math.floor(list.length / 2);
+//   const median = list.splice(middleIndex, 1)[0];
+//   if (middleIndex.length % 2 === 0) {
+//     return middleIndex[median - 1] + middleIndex[median] / 2;
+//   } else {
+//     return median;
+//   }
+// }
 function calculateMedian(list) {
-  const middleIndex = Math.floor(list.length / 2);
-  const median = list.splice(middleIndex, 1)[0];
-  return median;
+  const sortedList = [...list].sort((a, b) => a - b);
+  const median = Math.floor(sortedList.length / 2);
```

Comment:
wasn't the original variable name more descriptive ? ;)

Reply:
You're right that the original variable name middleIndex is more descriptive in the context of finding the median